app/views/volunteers/_form.html.erb

```diff
@@ -67,42 +67,100 @@
     </div>
 <br>
   <h4>Volunteer Interests and Availability</h4>
-  <p><strong>Availability:</strong>  Select a day you're availabile.</p>
+  <p>Select your availability from the options below.</p>
   <!-- style to make multiple checkbox options and style into columns -->
+    <div class="checkbox-background">
     <div class="form-group row">
-      <div class="col-md-6">
-        <%= f.radio_button :availability, 'Sunday' %>
-        <%= f.label :availability, 'Sunday' %> <br>
-
-        <%= f.radio_button :availability, 'Monday' %>
-        <%= f.label :availability, 'Monday' %> <br>
-                
-        <%= f.radio_button :availability, 'Tuesday' %>
-        <%= f.label :availability, 'Tuesday' %> <br>    
-         
-        <%= f.radio_button :availability, 'Wednesday' %>
-        <%= f.label :availability, 'Wednesday' %> <br>
-
-        <%= f.radio_button :availability, 'Thursday' %>               
-        <%= f.label :availability, 'Thursday' %> <br>
-            
-        <%= f.radio_button :availability, 'Friday' %>
-        <%= f.label :availability, 'Friday' %> <br>
-
-        <%= f.radio_button :availability, 'Saturday' %>
-        <%= f.label :availability, 'Saturday' %>
+      <div class="col"></div>
+      <div class="col"> 
+        <p>Sunday</p>
+      </div>
+      <div class="col">
+        <p>Monday</p>
+      </div>
+      <div class="col">
+        <p>Tuesday</p>
+      </div>
+      <div class="col">
+        <p>Wednesday</p>
+      </div>
+      <div class="col">
+        <p>Thursday</p>
+      </div>
+      <div class="col">
+        <p>Friday</p>
+      </div>
+      <div class="col">
+        <p>Saturday</p>
+      </div>
+    </div>
+    <div class="form-group row">
+      <div class="col">
+        <p>Mornings</p>
+      </div>
+      <div class="col checkbox">
+        <%= check_box("volunteer", "avail_sun_am",{},"Sunday Mornings", nil) %>
+      </div>
+      <div class="col checkbox">
+        <%= check_box("volunteer", "avail_mon_am",{},"Monday Mornings", nil) %>
```

Comment:
To clean up the code here a bit, you can consider returning all of these parameters inside of an array: https://apidock.com/rails/ActionView/Helpers/FormHelper/check_box#1001-Sending-array-parameters.